README.md

```diff
@@ -5,10 +5,14 @@ This repo builds off of the [official OCSF Java Tools](https://github.com/ocsf/o
 If you have already cloned the repository, you can build the docker image with:
 
 ```
+# Mount this repository in an openjdk container and invoke gradlew to build relevant .jar's
+sudo docker run --workdir=/app --volume $(pwd):/app openjdk:11 ./gradlew build
```

Comment:
I wasn't able to build without this line because all of the directories were missing the `build/libs/` subdirectories. Adding this `run` command let me build the missing `jar`s, which let me build the image.

A nice benefit here is that users without `java` on their local systems can still make changes to the java source and rebuild `ocsf-cli`, as long as they've got docker installed.